### PR TITLE
Use "organizations" not "common" as the default authority

### LINKED ISSUE #1366: [Issue] Login not possible after upgrade to 0.5.0
- [X] Make sure you've installed the latest version using [instructions in the wiki](../wiki/install)

**Output from `azd version`**
Run `azd version` and copy and paste the output here:

azd version 0.5.0-beta.1 (commit 57cba771d598398f30acfef18a272d1906a8e4f1)

**Output from `az version`** 
Run `az version` and copy and paste the output here (minimum required version is 2.38.0): 

{
  "azure-cli": "2.39.0",
  "azure-cli-core": "2.39.0",
  "azure-cli-telemetry": "1.0.6",
  "extensions": {}
}

**Describe the bug**

After the upgrade to 0.5.0-beta.1 I use the new login flow via `azd login`. The login option "Interactive authentication with a browser" is started opening a browser window.

Authentication via username (=email) states that:

```bash
This username may be incorrect. Make sure you typed it correctly. Otherwise, contact your admin.
``` 

A login with this username via `az login` or via Azure Portal is working. 

To work around this issue, I did:

1. `azd config set auth.useAzCliAuth true` 

which forces `azd` to use the login credential saved by Azure CLI (after running `az login`).

**To Reproduce**

- Install/upgrade to latest version of Azure Developer CLI
- Start the command `azd login` to kick off the "Interactive authentication with a browse"
- Enter your username and press "next"

**Expected behavior**

The interactive login via browser and username should be possible

**Environment**

Windows 11
PowerShell 7.3.1

**Additional context**

- Screenshot of error message:

   Screenshot 2023-01-12 161502

- The fallback via `azd config set auth.useAzCliAuth true` works

### PR DESCRIPTION
**Pull request by ellismg**
Organizations supports using both work/school accounts as well as personal accounts. This matches the default that the `az` CLI uses when logging in.

Fixes #1366

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]

File: cli/azd/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 0.5.1-beta.1 (2023-01-12)
```
Comment (weikanglim) [inline cli/azd/CHANGELOG.md]: In the past, we've been bumping the `-beta.X` segment of the pre-release version. Are we bumping `.X-beta.0` to highlight the bigger change instead?

File: cli/azd/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+
+### Bugs Fixed
+
+- [[#1366]](https://github.com/Azure/azure-dev/issues/1366) Login not possible after upgrade to 0.5.0.
```
Comment (weikanglim) [inline cli/azd/CHANGELOG.md:7]:
```suggestion
- [[#1366]](https://github.com/Azure/azure-dev/issues/1366) Login not possible with personal account after upgrade to 0.5.0.
```

**Review by weikanglim — APPROVED** [review]

File: cli/version.txt [hunk 3]
```diff
@@ -1 +1 @@
-0.5.0-beta.1
+0.5.1-beta.1
```
Comment (weikanglim) [inline cli/version.txt:1]: Noting in case this is unintentionally left as 0.5.0-beta.2

File: cli/azd/CHANGELOG.md (hunk 1 above)
Comment (ellismg) [reply cli/azd/CHANGELOG.md]: Went back to follow the previous pattern as you suggested.  This is now 0.5.0-beta.2

**Review by wbreza — COMMENTED** [review]
LGTM

**Pull request merged**